internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,444 @@
+	CleanupPolicy                  string `yaml:"cleanup.policy"`
+	DeleteRetentionMs              int    `yaml:"delete.retention.ms"`
+	ConfluentValueSchemaValidation string `yaml:"confluent.value.schema.validation"`
+}
+
+type KafkaBinding struct {
+	XConfigs map[string]string `yaml:"x-configs"`
+}
+
+type Message struct {
+	SchemaFormat string      `yaml:"schemaFormat"`
+	ContentType  string      `yaml:"contentType"`
+	Payload      interface{} `yaml:"payload"`
+	Name         string      `yaml:"name"`
+	Tags         []spec2.Tag `yaml:"tags"`
+}
+
+type Spec struct {
+	Asyncapi   string           `yaml:"asyncapi"`
+	Info       spec2.Info       `yaml:"info"`
+	Servers    spec2.Server     `yaml:"servers"`
+	Channels   map[string]Topic `yaml:"channels"`
+	Components Components       `yaml:"components"`
+}
+
+type Components struct {
+	Messages        map[string]Message `yaml:"messages"`
+	SecuritySchemes interface{}        `yaml:"securitySchemes"`
+}
+
+type Security struct {
+	ConfluentBroker         []any `yaml:"confluentBroker"`
+	ConfluentSchemaRegistry []any `yaml:"confluentSchemaRegistry"`
+}
+
+type MessageRef struct {
+	Ref string `yaml:"$ref"`
+}
+
+type Operation struct {
+	OperationID string                        `yaml:"operationId"`
+	TopicTags   []spec2.Tag                   `yaml:"tags"`
+	OpBindings  spec2.OperationBindingsObject `yaml:"bindings"`
+	Message     MessageRef                    `yaml:"message"`
+}
+
+type Topic struct {
+	Description           string        `yaml:"description"`
+	Publish               Operation     `yaml:"publish"`
+	Subscribe             Operation     `yaml:"subscribe"`
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	TopicBinding          TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka KafkaBinding `yaml:"kafka"`
+}
+
+type ConfluentBrokerXConfigs struct {
+	SaslMechanisms   string `yaml:"sasl.mechanisms"`
+	SaslPassword     string `yaml:"sasl.password"`
+	SaslUsername     string `yaml:"sasl.username"`
+	SecurityProtocol string `yaml:"security.protocol"`
+}
+
+type ConfluentBroker struct {
+	Type     string                  `yaml:"type"`
+	XConfigs ConfluentBrokerXConfigs `yaml:"x-configs"`
+}
+
+type ConfluentSRXConfigs struct {
+	BasicAuthUserInfo string `yaml:"basic.auth.user.info"`
+}
+type ConfluentSchemaRegistry struct {
+	Type     string              `yaml:"type"`
+	XConfigs ConfluentSRXConfigs `yaml:"x-configs"`
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import <spec.yaml>",
+		Short: "Adds topics, schemas and tags from the yaml file.",
+		Args:  cobra.ExactArgs(1),
+		Example: examples.BuildExampleString(
+			examples.Example{Text: "Import an asyncapi specification file to populate an existing cluster",
+				Code: "confluent asyncapi import spec.yaml",
+			},
+		),
+	}
+	c := &command{pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner), nil}
+	cmd.RunE = c.parse
```

Comment:
Function should have the same name as the command
```suggestion
	cmd.RunE = c.import
	
```

Reply:
import is a keyword in Go and cannot be used for naming a function